forem/forem
# Admin member area a11y fixes: Overview page

## What type of PR is this? (check all applicable)


## Description

This PR encompasses some minor fixes to ensure accessibility of the member detail view, focusing first on the Overview tab.  

_There are a couple of other places where IDs are duplicated and therefore labels don't get correctly associated, but I think these are all fixed in Nicks E2E tests PR: https://github.com/forem/forem/pull/16513. I'm going to keep this PR in draft until that's merged and I can update from it, as I think we have a few other areas of overlap too._
 
## Related Tickets & Documents

Contributes toward https://github.com/forem/forem/issues/16504

## QA Instructions, Screenshots, Recordings

- Check out the new member detail view locally - it should all look the same, with the exception of the "tags moderated", which should no longer have the 'X' icon (since a user can't delete them from this page).
- Inline with the code changes I've added some notes on what's changed and why, feel free to check those areas with a screen reader switched on, or e.g. inspect input labels/descriptions in dev tools

### UI accessibility concerns?

Addressing several small accessibility issues

## Added/updated tests?

- [X] No, and this is why: E2E tests are currently being written separately. I'll update this branch once they're merged to main, and make sure I make any necessary changes.

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_
